src/fields/mod.rs

```diff
@@ -139,22 +162,20 @@ pub trait CodecFieldElement:
     }
 }
 
-/// Elements of a field in which we can interpolate polynomials up to degree two. Our nodes are
-/// `x_0 = 0`, `x_1 = 1`, and `x_2 = SUMCHECK_P2` in the field. Since we only have three nodes, we
-/// can work out each Lagrange basis polynomial by hand. We precompute the denominators to avoid
-/// implementing division.
-///
-/// For details see [Section 6.6][1] and [2].
+/// Field elements used directly in proofs.
 ///
-/// # Bugs
+/// This trait provides methods to interpolate polynomials in two different contexts. For the
+/// Sumcheck sub-protocol, [`ProofFieldElement::lagrange_basis_polynomial_0()`],
+/// [`ProofFieldElement::lagrange_basis_polynomial_1()`], and
+/// [`ProofFieldElement::lagrange_basis_polynomial_2()`], provide the basis polynomials necessary to
+/// interpolate degree two polynomials. For the Ligero sub-protocol, [`ProofFieldElement::extend()`]
+/// performs Reed-Solomon encoding.
```

Comment:
Let's put the links to details closer to the concepts:
```suggestion
/// interpolate degree two polynomials (see [Section 6.6][1] and [2] for details). For the Ligero sub-protocol, [`ProofFieldElement::extend()`]
/// performs Reed-Solomon encoding (see Section 3.2 of [3](https://eprint.iacr.org/2024/2010.pdf) for details).
```